python/statistics_comparison.py

```diff
@@ -0,0 +1,169 @@
+              "git remote add " + repo.name + " " + repo.url + " && " + \
+              "git remote update && " + \
+              "git fetch " + repo.name + " && " + \
+              "git checkout -b test " + repo.name + "/" + repo.branch + " && " + \
+              "cmake .. && " + \
+              "make install && " + \
+              "cd /workdir/cvmfs/test && " + \
+              "./run.sh /tmp/benchmarks.log " + tests_to_execute + "\'"
+
+        print("Executing benchmarks for the repository \"" + repo.url +
+              "\"" + " in the branch \"" + repo.branch + "\"")
+        container_id = client.create_container(image=image + ":" + tag,
+                                               volumes=volumes,
+                                               hostname="cvmfs-test",
+                                               tty=True,
+                                               command=cmd).get("Id")
+        client.start(container_id, privileged=True, binds=binds)
+        result = client.wait(container_id)
+        if result != 0:
+            print("Test execution failed! Exiting...")
+            exit(100)
+
+
+def parse_args():
+    argparser = argparse.ArgumentParser()
+    argparser.add_argument("--socket_url",
+                           default="unix://var/run/docker.sock",
+                           required=False, type=str,
+                           help="""Docker socket to listen to.
+                                It has to be in one of the following ways:
+                                unix://path/to/docker.sock
+                                tcp://X.X.X.X:port""")
+    argparser.add_argument("--docker_api_version",
+                           default="1.17",
+                           required=False, type=str,
+                           help="Docker API version (default 1.17)")
+    argparser.add_argument("--benchmarks",
+                           default="atlas",
+                           required=False, type=str,
+                           help="""Comma-separated list of benchmarks.
+                           For example:
+                           atlas,lhcb,alice,cms
+                           cms,lhcb""")
+    argparser.add_argument("--original_repo",
+                           default="https://github.com/cvmfs/cvmfs.git",
+                           required=False, type=str,
+                           help="Original repository to compare to")
+    argparser.add_argument("--original_branch",
+                           default="devel",
+                           required=False, type=str,
+                           help="Original branch to compare to")
+    argparser.add_argument("--external_branch",
+                           default="devel",
+                           required=False, type=str,
+                           help="External branch to compare to")
+    argparser.add_argument("external_repo", type=str)
+    return argparser.parse_args()
+
+
+def get_benchmark_list(benchmark_string):
```

Comment:
Would it be possible to make this in a way that doesn't require knowledge about the list of benchmarks?  Essentially I would provide `benchmarks/XYZ-mybenchmark` which you wouldn't need to further parse or modify,